cPP_DSC.adoc

```diff
@@ -2673,16 +2725,16 @@ FCS_CKM.5.1:: The TSF shall derive cryptographic keys with [*selection*: _key ty
 |Shared secret, salt, IV, output length, fixed information 
 |[MAC Step]
 
-{empty}[*selection*: [.underline]#AES-128-CMAC; AES-192 -CMAC; AES-256 -CMAC; HMAC-SHA-1; HMAC-SHA-256; HMAC-SHA-512]# as the EEF and;
```

Comment:
The term `EEF` ("entropy extraction function"?) is not defined in this document. I think it would be easiest to just replace it with `MAC`.

Reply:
#308